app/feature/feature-forever/src/main/kotlin/com/hedvig/android/feature/forever/ui/DiscountPieChart.kt

```diff
@@ -0,0 +1,100 @@
+package com.hedvig.android.feature.forever.ui
+
+import androidx.compose.animation.core.FastOutSlowInEasing
+import androidx.compose.animation.core.InfiniteRepeatableSpec
+import androidx.compose.animation.core.KeyframesSpec
+import androidx.compose.animation.core.RepeatMode
+import androidx.compose.animation.core.animateFloat
+import androidx.compose.animation.core.rememberInfiniteTransition
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.unit.dp
+import com.hedvig.android.core.designsystem.material3.typeElement
+import com.hedvig.android.core.designsystem.preview.HedvigPreview
+import com.hedvig.android.core.designsystem.theme.HedvigTheme
+
+@Composable
+fun ColumnScope.DiscountPieChart(
+  containerColor: Color = MaterialTheme.colorScheme.typeElement,
+  discountColor: Color = MaterialTheme.colorScheme.secondaryContainer,
+  incentiveColor: Color = MaterialTheme.colorScheme.surface,
```

Comment:
Not sure about these colors, I think we want a darker green for the discount color here